Assets/Scripts/SS3D/Systems/Storage/Items/ItemSystem.cs

```diff
@@ -33,22 +29,18 @@ protected override void OnStart()
         /// <summary>
         /// Loads the item prefabs into memory with the item id and setting up the item ids variable in the items.
         /// </summary>
-        private async void LoadItemPrefabs()
```

Comment:
what was the async for and why it's gone ?

Reply:
Was used for the Addressables stuff, now we don't use it.